.github/fabricbot.json

```diff
@@ -1,939 +0,0 @@
-              }
-            },
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Completed"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Documentation Approved Label",
-        "actions": [
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "pending documentation"
-            }
-          },
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "documentation approved"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "UWzHHj6F95GSzUcxTPeY3",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "hasLabel",
-                  "parameters": {
-                    "label": "proposal"
-                  }
-                },
-                {
-                  "name": "isInProject",
-                  "parameters": {
-                    "projectName": "New Feature Proposals"
-                  }
-                },
-                {
-                  "name": "titleContains",
-                  "parameters": {
-                    "titlePattern": "[Proposal]"
-                  }
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isOpen",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "operator": "or",
-                  "operands": [
-                    {
-                      "name": "isInProjectColumn",
-                      "parameters": {
-                        "projectName": "New Feature Proposals",
-                        "columnName": "Closed"
-                      }
-                    },
-                    {
-                      "name": "isInProjectColumn",
-                      "parameters": {
-                        "projectName": "New Feature Proposals",
-                        "columnName": "Completed"
-                      }
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Reopen Closed Proposals Not In Closed Column",
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.